Author: Daniel Baumann <daniel@debian.org>
Description:
 Replacing adddir location with absolute name in order to allow to compile the
 themes from any location, not just /usr/share/gfxboot/themes/foo.

--- a/themes/openSUSE/Makefile
+++ b/themes/openSUSE/Makefile
@@ -10,7 +10,7 @@ else
 PREPARED     = $(shell [ -f .prepared ] && echo 1)
 endif
 
-ADDDIR       = ../../bin/adddir
+ADDDIR       = /usr/share/gfxboot/bin/adddir
 BFLAGS       = -O -v -L ../..
 
 SUBDIRS      = fonts help-boot help-install po src
